cni-genie_k8s_test.go

```diff
@@ -188,7 +188,7 @@ var _ = Describe("CNIGenie", func() {
 
 			It("should succeed multi-ip networking for pod", func() {
 				annots := make(map[string]string)
-				annots["cni"] = "calico,weave"
+				annots["cni"] = "flannel,weave"
```

Comment:
@sushanthakumar
For some reason when I passed "weave,calico", calico failed! When I pass "calico" only it succeeds. Have you come across this issue before?

Reply:
I've just noticed this comment. calico needs to be a first on the list. calico fails when a default route is already configured and probably weave is configuring a default route.